articles/cognitive-services/Face/Tutorials/FaceAPIinJavaForAndroidTutorial.md

```diff
@@ -133,49 +138,60 @@ The Face API is a cloud API which you can invoke using HTTPS requests. For a mor
 
 Follow the instructions below to configure the client library: 
 
-1. Locate the top-level build.gradle file of your project from the Project panel shown in the example. Note that there are several other build.gradle files in your project tree, and you need to open the top-level build.gradle file at first. 		 
-2. Add mavenCentral() to your projects' repositories. You can also use jcenter(), which is the default repository of Android Studio, since jcenter() is a superset of mavenCentral().  
+1. Locate the top-level **build.gradle** file of your project from the Project panel shown in the example. Note that there are several other **build.gradle** files in your project tree, and you need to open the top-level **build.gradle** file at first. 		 
+2. Add **mavenCentral()** to your projects' repositories. You can also use jcenter(), which is the default repository of Android Studio, since jcenter() is a superset of mavenCentral().  
 
-		allprojects {
-		    repositories {
-		        ...
-		        mavenCentral()
-		    }
+```
+	allprojects {
+		repositories {
+			...
+			mavenCentral()
 		}
+	}
+```
 
-3. Open the build.gradle file in your 'app' project.
+3. Open the **build.gradle** file in your 'app' project.
 4. Add a dependency for our client library stored in the Maven Central Repository:
 
-		dependencies {  
-		    ...  
-		    compile 'com.microsoft.projectoxford:face:1.0.0'  
-		}  
+```
+	dependencies {  
+		...  
+		compile 'com.microsoft.projectoxford:face:1.4.2'  
```

Comment:
1. There are some issues for version 1.4.2, let us make it 1.4.3 for convenience. The version 1.4.3 is just released yesterday. :-)
2. Due to the new rules of `build.gradle`, we prefer to use `implementation` rather than `compile` now.